src/target/efm32.c

```diff
@@ -349,79 +343,49 @@ typedef struct efm32_v2_di_miscchip_t {
 /* pkgtype */
 typedef struct efm32_v2_di_pkgtype_t {
 	uint8_t pkgtype;
-	char* name;
+	char *name;
 } efm32_v2_di_pkgtype_t;
 
 efm32_v2_di_pkgtype_t const efm32_v2_di_pkgtypes[] = {
-	{74, "WLCSP"},	/* WLCSP package */
-	{76, "BGA"},	/* BGA package */
-	{77, "QFN"},	/* QFN package */
-	{81, "QFxP"},	/* QFP package */
+	{74, "WLCSP"}, /* WLCSP package */
+	{76, "BGA"},   /* BGA package */
+	{77, "QFN"},   /* QFN package */
+	{81, "QFxP"},  /* QFP package */
 };
 
 /* tempgrade */
 typedef struct efm32_v2_di_tempgrade_t {
 	uint8_t tempgrade;
-	char* name;
+	char *name;
 } efm32_v2_di_tempgrade_t;
 
 efm32_v2_di_tempgrade_t const efm32_v2_di_tempgrades[] = {
 	{0, "-40 to 85degC"},
 	{1, "-40 to 125degC"},
 	{2, "-40 to 105degC"},
-	{3, "0 to 70degC"}
+	{3, "0 to 70degC"},
 };
 
 /* -------------------------------------------------------------------------- */
-/* Helper functions - Version 1 V1 */
+/* Helper functions                                                           */
 /* -------------------------------------------------------------------------- */
 
-/**
- * Reads the EFM32 Extended Unique Identifier EUI64 (V1)
- */
+/* Reads the EFM32 Extended Unique Identifier EUI64 (V1) */
 static uint64_t efm32_v1_read_eui64(target *t)
 {
-	uint64_t eui;
-
-	eui  = (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_1) << 32;
-	eui |= (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_0) <<  0;
-
-	return eui;
+	return (uint64_t)target_mem_read32(t, EFM32_V1_DI_EUI64_1) << 32 | target_mem_read32(t, EFM32_V1_DI_EUI64_0);
```

Comment:
combining this is fine but that makes the ()'s around the shift expression required due to operator precedence.

Reply:
oh, right.. it does take precedence over `|`, just not `||` - still would ask to have them added as it's then consistent with the rest of the code base and allays these kinds of confusion

Reply:
yep, that is fine